WordPress/gutenberg
# Bootstrap the widgets screen design

Related #13204 

<img width="1281" alt="Capture d’écran 2019-03-25 à 9 42 19 AM" src="https://user-images.githubusercontent.com/272444/54905945-77bd9780-4ee2-11e9-8e0f-6ee170ac8da7.png">

This PR bootstraps the widgets screen design:

 - Saving/fetching... doesn't work
 - It uses a static list of areas
 - It cherry-picks #14369 in order to have separate states for each widget area
 - It uses the block editor module
 - It mimics the aria region navigation of the edit post screen

This PR doesn't intent to solve all the issues related to this screen and is intended to land a first dumb screen in order to allow us to parallelize work and start tackling all the related issues.

**Testing instructions**

 - Navigate to `/wp-admin/admin.php?page=gutenberg-widgets`
 - You should see what ressembles the screenshot above.
